index.html

```diff
@@ -160,73 +227,89 @@ <h1>About</h1>
-                            placeholder="Your email">
-                        <textarea name="subject" id="msg" placeholder="Enter your message here..." maxlength="500" required></textarea>
-                        <input type="submit" value="Start collaboration" id="button">
-                    </form>
-                </section>
-
-                    <footer class="myFooter">
-                        <h3>Get my Resume <img src="./images/footerImage.png" alt="logo"></h3>
-                        <ul class="myHandles">
-                            <li><a href="https://stackoverflow.com/"><img src="./images/ic_stackoverflow.png" alt="stackflow logo"></a></li> 
-                            <li><a href="https://github.com/solorics"><img src="./images/github.png" alt="github logo"></a></li>
-                            <li><a href="https://twitter.com/Solorics2"><img src="./images/skype.png" alt="twitter logo"></a></li>
-                            <li><a href="https://www.linkedin.com/in/solomon-opeyemi-0427a6155/"><img src="./images/linkdeln.png" alt="linkdeln logo"></a></li>
-                            <li><img src="./images/microverse.png" alt="microverse logo"></li>
-                            <li><img src="./images/twitter2.png" alt="twitter logo"></li>
+                    <div class="aboutcard3">
+                        <a class="heading3" href="#">Skills</a>
+                        <ul class="link3">
+                            <li>Codekit</li>
+                            <li>Codepen</li>
+                            <li>GitHub</li>
+                            <li>GitLAb</li>
+                            <li>Terminal</li>
+                            <img src="./images/ic_skills.png" alt="this is a
+                                image" id="bulb">
                         </ul>
-                    </footer>
-            </div>
-        </body>
\ No newline at end of file
+                    </div>
+                </div>
+            </section>
+
+            <section class="contactForm">
+                <div class="contactLogo"></div>
+                <div class="content">
+                    <h1>Get started</h1>
+                    <p>Please get in touch and our expert support team will
+                        answer all your questions.</p>
+                </div>
+                <form action="https://formspree.io/f/mjvzkrzj" method="POST">
+                    <input type="text" required id="fullName" placeholder="Your
+                        name"
+                        name="Full Name" maxlength="30">
+                    <input type="email" required name="Email" id="myEmail"
+                        placeholder="Your email">
+                    <textarea name="subject" id="msg" placeholder="Enter your
+                        message here..." maxlength="500" required></textarea>
+                    <input type="submit" value="Start collaboration"
+                        id="button">
+                </form>
```

Comment:
- [ ] There is a little difference between the mobile version and the desktop version, it is the labels above the inputs. Please 🙏🏽 add them for a better user interface and correct the position of the placeholders.

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/199863142-44940c6c-4b53-4685-923c-9ab9755eb3b0.png) | ![image](https://user-images.githubusercontent.com/58642949/199863215-8362e255-40d4-4437-901f-7d9b5e3422fd.png) |

Reply:
Thanks. 🙏 
I have worked on it.